add sample for aurora serverless v2 scheduled scaling

jhart0 opened the pull request:
Moves samples to samples directory. 

Adds new sample for scheduled auto-scaling of Aurora Serverless V2. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

nihilg commented:
1. In the README.md > Contents >
The link to Scheduled Autoscaling Aurora Serverless V2
Is broken and Is pointing to : https://github.com/aws-samples/data-for-saas-patterns/blob/feat/scheduled- scaling-apgv2/README.md#**scheduling**-autoscaling-aurora-serverless-v2
The link should resolve to : https://github.com/aws-samples/data-for-saas-patterns/blob/feat/scheduled- scaling-apgv2/README.md#**scheduled**-autoscaling-aurora-serverless-v2

[Scheduled Autoscaling Aurora Serverless V2](README.md#**scheduled**- autoscaling-aurora-serverless-v2)

2. The samples/multi-tenant-vector-database/amazon-aurora/cdk/ README.md file needs update as well to include the samples sub- directory.
Deployment Instructions
Step 2
cd data-for-saas-patterns/multi-tenant-vector-database/amazon-aurora/ cdk
3. Confirm validating the NAG rules for Lambda in the CDK project ( aurora serverless v2 scheduled scaling)
4. It was difficult for me to identify the dbClusterId for the Aurora Serverless cluster needs to be set in the cdk.context.json ( Just a comment and No fix required)
5. In the “scheduled-aurora-serverless-scaling” sample readme, post deployment instructions there was no mention of how to validate the scheduled scaling has run based on the defined cron parameters. I could not find a CLI command to read the scaling configuration and the current ACU. Looks like cloud watch metrics are the only way to monitor. Do you want to hint how to validate ?

jhart0 commented:
Thanks @nihilg ! 

I've addressed all your comments in the latest commit: https://github.com/aws-samples/data-for-saas-patterns/pull/3/commits/94c7b90893c8960cd05c649c03b5506895c21db2

nihilg commented:
+ Review complete !!
